airgun/views/contentview.py

```diff
@@ -210,3 +326,123 @@ def is_displayed(self):
                 and self.breadcrumb.locations[0] == 'Content Views'
                 and self.breadcrumb.read() == 'Promotion'
         )
+
+
+class ContentViewVersionRemoveView(BaseLoggedInView):
+    breadcrumb = BreadCrumb()
+
+    table = SatTable(
+        locator='.//table',
+        column_widgets={
+            0: Checkbox(locator="./input[@type='checkbox']"),
+        }
+    )
+    completely = Checkbox(
+        locator=".//input[@ng-model='deleteOptions.deleteArchive']")
+    next = Text(".//button[@ng-click='processSelection()']")
+    cancel = Text(".//button[normalize-space(.)='Cancel']")
+
+    @property
+    def is_displayed(self):
+        breadcrumb_loaded = self.browser.wait_for_element(
+            self.breadcrumb, exception=False)
+        return (
+                breadcrumb_loaded
+                and self.breadcrumb.locations[0] == 'Content Views'
```

Comment:
does it make sense to have so many conditions? Here and previously

Reply:
Yep, unfortunately that's special case when we have multiple different CV pages with the same breadcrumb location, so we can't just rely on breadcrumb only to figure out where we are.

Reply:
okay